docs/source/chatops/index.rst

```diff
@@ -50,6 +50,7 @@ Interested in learning more? Here are some things to get you started on your voy
     :maxdepth: 1
 
     chatops
+    Configuration for Microsoft Teams <msteams_is_a_diva>
```

Comment:
Perhaps we should have an "Adapters" section where we list supported adapters, their configuration option and limitations?

Reply:
Excellent idea - done!

Reply:
Well, somewhat done. I don't list the exact options and limitations, but I do point people to the source code so they can figure out what to do themselves. Since this is more of an advanced option, I'm not too worried about not having perfect documentation for it.